include/TPP/Dialect/Xsmm/XsmmOps.td

```diff
@@ -263,6 +263,15 @@ def Xsmm_GemmDispatchOp : Xsmm_GemmLikeOp<"gemm.dispatch"> {
 def Xsmm_BrgemmDispatchOp : Xsmm_GemmLikeOp<"brgemm.dispatch"> {
   let summary = "dispatch for brgemm operation.";
   let hasVerifier = 1;
+  
+}
+
+//===----------------------------------------------------------------------===//
+// TileConfigDispatchOp
+//===----------------------------------------------------------------------===//
+
+def Xsmm_TileConfigDispatchOp : Xsmm_GemmLikeOp<"tileConfig.dispatch"> {
```

Comment:
It'd be best to have a separate base class or generalize the `Xsmm_GemmLikeOp` if it fits here.

Reply:
Without any libxsmm API knowledge, tile config sounds generic. So, does it configure/affect other operations too?
If not, then maybe GEMM (tile) config would be more accurate and at least that would fit the current inheritance scheme.

I just don't wanna further the spaghetti like the fused brgemm which already is not a GemmLikeOp for whatever reason.
Overall, I'd need better op documentation or at least to see some IR example to have any concrete suggestions.

Reply:
It is a 1-to-1 copy of the libxsmm API. Once we improve the API, we can change it here